### PR TITLE
Update common files

### PR DESCRIPTION
**Pull request by julienbourdeau**
:gift:

### CODE REVIEW

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2013 Algolia
-http://www.algolia.com/
+Copyright (c) 2014-Present Algolia
```
Comment (PLNech) [inline LICENSE]: All in for `-Present`, but why moving from 2013 to 2014? :thinking:

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## `algolia/algoliasearch-client-python` maintainers
+
+| Name             | Email                       |
+|------------------|-----------------------------|
+| Léo Ercolanelli | leo.ercolanelli@algolia.com |
+| Paul-Louis Nech  | support@algolia.com         |
```
Comment (PLNech) [inline MAINTAINERS.md]: AFAIK I'm not the maintainer of this package ¯\_(ツ)_/¯

File: LICENSE (hunk 1 above)
Comment (julienbourdeau) [reply LICENSE]: All clients use the same Licence file, I guess Python is one of the older. I'll how ot change that

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Pull request merged**